app/bundles/LeadBundle/Entity/OperatorListTrait.php

```diff
@@ -179,27 +179,25 @@ public function getOperatorChoiceList($definition, $overrideHiddenOperators = []
 
     /**
      * @deprecated These aliases are subscribed in the TypeOperatorSubscriber now so this is not necessary. To be removed in next Mautic version.
-     *
-     * Normalize type operator.
-     *
-     * @param string|null $type
      */
-    protected function normalizeType(&$type)
+    protected function normalizeType(mixed $type): mixed
     {
         if (null === $type) {
-            return;
+            return null;
         }
 
         if ('boolean' === $type) {
-            $type = 'bool';
+            return 'bool';
         } elseif (in_array($type, ['country', 'timezone', 'region', 'locale'])) {
-            $type = 'select';
+            return 'select';
         } elseif (in_array($type, ['lookup',  'text', 'email', 'url', 'email', 'tel'])) {
-            $type = 'text';
+            return 'text';
         } elseif ('datetime' === $type) {
-            $type = 'date';
+            return 'date';
         } elseif (!array_key_exists($type, $this->typeOperators)) {
-            $type = 'default';
+            return 'default';
         }
+
+        return $type;
```

Comment:
I've refactored this from reference to return value, so we can add `mixed` return and param type :+1: